core/src/main/resources/reference.conf

```diff
@@ -295,6 +295,10 @@ filesystems {
       config {
         martha {
           url = "https://martha-url-here"
+          # When true (and when possible) any DrsPath will be internally converted to another compatible Path such as a
+          # GcsPath. This enables other optimizations such as using batch requests for GCS for hashes, using the
+          # existing GCS downloader in Papi, etc.
+          preresolve = false
```

Comment:
So this will be `true` when we configure cromwell in terra?

Reply:
Could be. Maybe we’ll only turn it on for HCA testing in Terra-Dev and then if that works turn it on in the other Terra environments. Or others could test it in say the methods Cromwells while they also work on various DOS/DRS compatibility work.